### PR TITLE
Update Done checklist for Product Backlog Item

### PR DESCRIPTION
**Pull request by rija**
Result of long discussions a while ago:

* [ ] Item(s) in increment pass all Acceptance Criteria
* [ ] Code is refactored to best practices and coding standards
* [ ] Documentation is updated as needed
* [ ] Data security has not be compromised (with particular reference to the personal information we hold in GigaDB)
* [ ] No deviation from the team technology stack and software architecture has been introduced
* [ ] The product is in a releasable state (I.e. the increment has not broken anything)

### CODE REVIEW

**Review by kencho51 — APPROVED** [review]
Hi @rija,

The changes look good to me.

File: .github/ISSUE_TEMPLATE/product_backlog_item.md [hunk 1]
```diff
@@ -42,14 +42,11 @@ assignees: ''
...
-* [ ] Automated tests related to the changes are implemented and passing
-* [ ] All automated test suites are passing locally
+* [ ] Item(s) in increment pass all Acceptance Criteria
 * [ ] Code is refactored to best practices and coding standards
 * [ ] Documentation is updated as needed
-* [ ] A Pull Request has been created and review requested
-* [ ] Pull Request is reviewed and approved
-* [ ] The item has been merged to the develop branch
-* [ ] All automated test suites are passing on continuous Integration pipeline and item is ready to release
+* [ ] Data security has not be compromised (with particular reference to the personal information we hold in GigaDB)
```
Comment (pli888) [inline .github/ISSUE_TEMPLATE/product_backlog_item.md:48]: Should be:

Data security has not `been` compromised

File: .github/ISSUE_TEMPLATE/product_backlog_item.md [hunk 2]
```diff
@@ -42,14 +42,11 @@ assignees: ''
...
+* [ ] No deviation from the team technology stack and software architecture has been introduced
+* [ ] The product is in a releasable state (I.e. the increment has not broken anything)
```
Comment (pli888) [inline .github/ISSUE_TEMPLATE/product_backlog_item.md:50]: I think `I.e.` should be `i.e.`.

**Review by pli888 — APPROVED** [review]
The changes look good to me - just check the typos.

**Comment by only1chunts** [comment]
typo : Data security has not **be** compromised -> Data security has not **been** compromised
otherwise it looks OK to me.

**Review by only1chunts — APPROVED** [review]

**Comment by rija** [comment]
Thanks everyone.
I've corrected the typos in the actual template file.

@pli888:

I don't have the permission to merge to the ``master`` branch, would you mind merging this for me?

**Pull request merged**